packages/formstr-app/src/containers/CreateFormNew/components/FormSettings/types.ts

```diff
@@ -1,11 +1,15 @@
+import { SectionData } from "../../providers/FormBuilder/typeDefs";
+
 export interface IFormSettings {
   titleImageUrl?: string;
   description?: string;
   thankYouPage?: boolean;
-  notifyNpubs?: Array<string>;
+  notifyNpubs?: string[];
   publicForm?: boolean;
   disallowAnonymous?: boolean;
+  encryptForm?: boolean;
+  viewKeyInUrl?: boolean;
   formId?: string;
-  encryptForm?: boolean
-  viewKeyInUrl?: boolean
-}
+  sections?: SectionData[];
+  enableSections?: boolean;
```

Comment:
can enableSections be deciphered by the presence of sections? in which case do we need to pass it as a prop?